gui/src/components/dashboard/dashboard.vue

```diff
@@ -57,11 +58,29 @@ export default class Dashboard extends Vue {
   public alertTblRowHeight: number = 0;
   public alertTblRowHeightPx: string = "";
   public chartRowHeightPx: string = "";
+  public colNumber: number = 8;
```

Comment:
Variable name can be a bit more descriptive like alertTableColWidthNumber

Reply:
Updated variable name